### DISCUSSION TITLE
Leistung parallel aus PV und Batterie und umgekehrt

### DISCUSSION THREAD

**Discussion by Technikfreak72** [hoylabs/OpenDTU-OnBattery#304] [Q&A] [answered]
Hallo,
tolles Projekt! OpenDTU ist bei mir mit einem HM1500 schon im Einsatz, HomeAssistant auch und ein Hichi.
Nun soll eine Batterie her.

1. Nehmen wir an, der Leistungsbedarf beträgt 300W, die Batterie ist ausreichend geladen, die PV Module liefern 200W. Werden dann die fehlenden 100W aus der Batterie entnommen?

2. Nehmen wir an, der Leistungsbedarf beträgt 400W, die PV liefert 500W. Wird die Batterie dann parallel mit 100W geladen?

**Comment by schlimmchen** [comment, marked as answer]
Verluste hast du nicht berücksichtigt, aber das Prinzip des Dynamic Power Limiter hast du verstanden. Hast du die `README_onBattery.mb` schon gründlich durchgelesen?

**Reply by Technikfreak72** [reply]
Danke für die schnelle Antwort . Ich habe alles gelesen, aber das kam nicht eindeutig raus. Die Anket Solix Solarbank kann nämlich genau das nicht, weshalb ich meine Bestellung bei Anker storniert habe.

**Comment by madmartin** [comment]
@Technikfreak72 Noch eines zu Bedenken:
Die Spannung der Batterie sollte 48V haben damit Du die Leistung des HM-1500 auch abrufen kannst. Bei 24V Batteriespannung funktioniert das nicht.

**Comment by Manos1966** [comment]
Hallo @Technikfreak72 vielleicht findest du diese Kommentare hilfreich (siehe unten). Ich habe sie in meinen Notizen, leider aber habe ich die Links zur ursprünglichen Diskussionen nicht:

> Du kannst eine Batterie an mehrere Anschlüsse anschliessen. 
Allerdings must du schauen, dass du deine Batterie nicht überlastest. D. h. es kommt auf die Leistung an, die du aus der Batterie ziehen darfst. Ferner zieht der Hoymiles meines Wissens maximal 10.5 A pro Anschluss. 
Bei einer 24 V Batterie sind es 24 V * 10,5A * 4 = 1008W D. h. du verschenkst fast 500W. 
Bei einer 48 V Batterie sind es 48 V * 10,5 A * 4 = 2016 W. D. h. du kannst den Inverter voll ausreizen.
(P.S. Mein Test mit einem HM-1500 hat 9,8A pro Anschluss max gezeigt, aber das kann an anderen Faktoren liegen )

> It's quite easy. Electrons always seek the path with the lowest resistance and lower voltage. 
**The inverter has a lower resistance (as long as it's pulling power) and will always be fed first and the remaining power goes into the battery.** 
That's true at least when the inverter, battery and charger are in the same curcuit without any components inbetween. 
The energy from the charger always takes priority because it has a higher voltage than the battery. So this energy flows either into the inverter or into the battery.

**Reply by Technikfreak72** [reply]
sehr gut, danke Euch! Jetzt bekomme ich richtig Lust auf das Projekt. HM-1500 und 4x415W mit Ost/West Ausrichtung sind schon in Betrieb. Daher hätte ich gerne 2 Victrons genommen, aber offensichtlich wird nur einer derzeit unterstützt

**Reply by spcqike** [reply]
Ist die Frage ob die die zwei tatsächlich brauchst. 

Dazu gibt es unterschiedliche Meinungen. Sicher sind unabhängige Pro Ausrichtung „besser“ (ernten mehr), aber eben in der Regel auch teurer. 

Man kann wohl auch nur einen laderegler nehmen. Dabei verliert man etwas Effizienz, allerdings ist die Investition weitaus geringer. 

Hier haben wir das schon einmal angesprochen und ich hab was dazu verlinkt. 

https://github.com/helgeerbe/OpenDTU-OnBattery/discussions/249#discussioncomment-6100464

**Reply by swingstate** [reply]
Ein MPPT langt durchaus - ich poste in ein paar Tagen ein Update mit Fotos.

**Reply by Technikfreak72** [reply]
der Verlust ist sicherlich nicht groß, aber ich denke auch an einen Ausbau später. Denn 2 kleine Victrons sind nicht wirklich teuer, als ein großer. Die Limitierung ist schade, aber dennoch ein super Projekt👍🏻
Im Moment experimentiere ich noch mit der Ecoflow Powerstream, aber die ist nicht wirklich geeignet für meine Anforderungen.

**Reply by Manos1966** [reply]
> Daher hätte ich gerne 2 Victrons genommen, aber offensichtlich wird nur einer derzeit unterstützt

Ich liebäugle selber zwei VICTRON 100/20 12/24/48V weil sie mit jeweils 2x2 Panels arbeiten koennen und viel preiswerter sind.
Ich habe bis jetzt die "Limitierung des nur einen VICTRON" als kein Problem gesehen:
Vorausgesetzt meine Batterie (gerade teste eine Pylontech 5000): 
a. sich selbst steuern kann, 
b. von VICTRON unterstutzt wird, und 
c. **die SOC Daten an die DTU übergibt**
und die VICTRON sich untereinander kommunizieren (**ich habe es so verstanden**, mehrere VICTRON MPTT 100/20 koennen untereinander ueber VE.Direkt angeschossen werden - **ich hoffe ich liege hier richtig**).

Bei Nutzung des Solar Passthrough entsteht vielleicht ein Nachteil: die OpenDTU sieht nur einen VICTRON, dadurch wird die Gesamtleistung aller VICTRONS nicht in Betracht gezogen - solange die Batterie nicht die Start-Spannung erreicht hat.
Hat die Batterie die Start Spannung erreicht, die OpenDTU steuert die Strom Production perfekt, also kein Problem.
Ein weiterer Nachteil ist, dass ich die Gesamtleistung aller VICTRONS nicht in der OpenDTU verfolgen kann - "Nice to have" aber nicht Tragisch...

Mein Verstaendnis nach ein paar Wochen im Forum ist, den SOC der Batterie verfolgen zu koennen ist viel besser als eine Steuerung auf Basis der Batteriespannung (vor allem wenn Tagsüber die VICTRON Strom _& Spannung_ produzieren).  Danach je mehr VICTRONS, desto schneller wird die Batterie geladen. 
Ein kleiner Verlust ist da (wenn die Extra VICTRONS erst die Batterie laden, anstatt Strom direkt an den Wechselrichter zu geben).
Den Vorteil aber, die Leistung weiterer 4 Panels zu haben um die Batterie schneller zu laden, finde ich unschlagbar!
Schliesslich heisst ja das Projekt "OpenDTUonBattery" und nicht "OpenDTUonVICTRON" ;-) 

Falls jemand ein Denkfehler in meiner Argumentation sieht, wurde ich gerne hoeren.

**Reply by spcqike** [reply]
> ich habe es so verstanden, mehrere VICTRON MPTT 100/20 koennen untereinander ueber VE.Direkt angeschossen werden - ich hoffe ich liege hier richtig

Wenn ich es richtig verstanden habe kommunizieren die Victrons per Bluetooth. Die smart haben es integriert, die Blue könnten ein dongle nehmen. 

Der Master gibt die Summe aller laderegler an. 

VE.Direkt ist nur ein Point-to-Point Protokoll für genau zwei Geräte. Kein Bus. 

Ich hoffe ich habe das so richtig heraus gelesen aus diversen Kommentaren :) weil sowas möchte ich später ggf. auch machen. 

Eventuell hat ja jemand so ein setup und kann uns erleuchten, ob in solch einem Bluetooth Verbund der Master alle Infos auf dem VE.Direkt an opendtu gibt oder nur seine eigenen Werte.

**Reply by Manos1966** [reply]
> Wenn ich es richtig verstanden habe kommunizieren die Victrons per Bluetooth. Die smart haben es integriert, die Blue könnten ein dongle nehmen.

Danke @spcqike du hast Recht. Ich habe dafuer sogar ein Video gefunden:
Laderegler Parallel anschließen VE.Smart Networking Victron
[https://www.youtube.com/watch?v=rkDQcIBGIUA](url)

Und (**_wenn ich richtig verstanden habe_**), man kann sogar unterschiedliche VICTRON MPTTs anschließen 👍 
[https://www.victronenergy.com/media/pg/VE.Smart_Networking/en/synchronised-charging---further-details.html](url)

> Der Master gibt die Summe aller laderegler an.

Bleibt nur diesen Punkt zu testen...
Falls niemand Infos darüber postet, werde ich es demnächst probieren, zur Zeit bin ich aber mit dem CANbus der Pylontech Batterie beschäftigt.
